metaflow/plugins/kfp/kfp.py

```diff
@@ -78,12 +85,15 @@ def create_kfp_pipeline_yaml(self, pipeline_file_path=DEFAULT_KFP_YAML_OUTPUT_PA
         )
         return os.path.abspath(pipeline_file_path)
 
-    @staticmethod
-    def _command(code_package_url, environment, step_name, step_cli):
+    def _command(self, code_package_url, environment, step_name, step_cli):
         """
         Analogous to batch.py
         """
-        commands = environment.get_package_commands(code_package_url)
+        commands = (
+            environment.get_package_commands(code_package_url)
+            if self.s3_code_package
+            else ["cd " + str(Path(inspect.getabsfile(self.flow.__class__)).parent)]
```

Comment:
So guessing the `s3_code_package` pieces are just to maintain the compatibility with the existing Metaflow patterns but we just intend to use the `base_image` which will contain the Code Package already?

Reply:
yes, the s3_code_package allows one to run `python hello.py --datastore=s3 kfp run` from their laptop onto KFP without needing to package hello.py into a Docker container!

example:
```bash
```